accept/urls.py

```diff
@@ -3,5 +3,6 @@
 
 urlpatterns = [
     url(r'^$', views.accept, name='accept-index'),
-    url(r'^process/$', views.accept_process, name='accept-process'),
+    url(r'^process/$', views.handover_process, name='handover-accept'),
```

Comment:
I think these names are getting confusing between accept/process/handover, and the naming isn't consistent within the app

`views.handover_process` is named `handover-accept`. should it be `handover-process`. Also, `views.accept` is named `accept-index`, but all the other views have names that start with `handover`.

Perhaps the app should be called `responsibility` or `ownership` or some other noun. Then actions like accept or reject make more sense?